objects/ConsoleExtension/Removable Media Security (secRMM) ConfigMgr console extension.json

```diff
@@ -1,6 +1,6 @@
 {
   "downloadLocation": "https://www.squadratechnologies.com/StaticContent/ProductDownload/secRMM/9.11.0.0/secRMMConfigMgrConsoleExtension.cab",
-  "FileHash": "C15D02F40173B313270656C300BCEC4ADCE9F8235587891561A21ECA415951FA",
+  "FileHash": "B54CBE5F261C74F4D932EE6D5FF6C7F97EB0FF5ADBA0DCD45EF3F8F35176449B",

```

Comment:
>FileHash [](http://example.com/codeflow?start=3&length=8)

This change looks fine, and the checks all pass, but we did notice that you didn't select the organization in the push update wizard.  Was that by design?

Reply:
Let us know if this was unintentional and we'll update in the background to put your org id back in place.

Reply:
Hi Mark,  sorry my mistake.  
Yes, could you please put the org id back in for me and I will make sure it is defined on my next update.
Thanks and have a good one!

Reply:
It has been restored, thanks!